homeassistant/components/telegram_bot/__init__.py

```diff
@@ -460,11 +456,12 @@ def initialize_bot(p_config):
     proxy_params = p_config.get(CONF_PROXY_PARAMS)
 
     if proxy_url is not None:
-        request = Request(
-            con_pool_size=8, proxy_url=proxy_url, urllib3_proxy_kwargs=proxy_params
-        )
+        # These have been kept for backwards compatibility, they can actually be stuffed into the URL.
```

Comment:
In the future we should deprecate these and raise a repair issue if we find them here.